{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "96889cc7_48b6680e",
        "filename": "ibm_vpd_app.cpp",
        "patchSetId": 9
      },
      "lineNbr": 351,
      "author": {
        "id": 1000362
      },
      "writtenOn": "2022-01-24T05:21:42Z",
      "side": 1,
      "message": "can you give a doxygen for this method to understand the purpose of it.",
      "revId": "87a4a2d83d89429c0a3963436aac76c083d83c8e",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "72c8c0bc_57e415a4",
        "filename": "ibm_vpd_app.cpp",
        "patchSetId": 9
      },
      "lineNbr": 381,
      "author": {
        "id": 1000522
      },
      "writtenOn": "2022-01-21T18:32:04Z",
      "side": 1,
      "message": "I am not 100% sure, still would like to know,\nwhat if the call is made from line 1440 and an exception is thrown by call method? Will that be caught in the next catch block or will it go uncaught and that is what we are expecting?",
      "revId": "87a4a2d83d89429c0a3963436aac76c083d83c8e",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "396d26e9_011f5819",
        "filename": "ibm_vpd_app.cpp",
        "patchSetId": 9
      },
      "lineNbr": 392,
      "author": {
        "id": 1000522
      },
      "writtenOn": "2022-01-21T18:32:04Z",
      "side": 1,
      "message": "Suggestion: \"call\" methohd will throw in all cases. So we can get rid of this check?\n\nhttps://gerrit.openbmc-project.xyz/c/openbmc/phosphor-virtual-sensor/+/38244",
      "range": {
        "startLine": 382,
        "startChar": 8,
        "endLine": 392,
        "endChar": 9
      },
      "revId": "87a4a2d83d89429c0a3963436aac76c083d83c8e",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "2fbecc98_3fdaabae",
        "filename": "ibm_vpd_app.cpp",
        "patchSetId": 9
      },
      "lineNbr": 421,
      "author": {
        "id": 1000362
      },
      "writtenOn": "2022-01-24T05:21:42Z",
      "side": 1,
      "message": "this returns true if the pcie is on pass 1 planar. but your cout statement is contradicting",
      "range": {
        "startLine": 421,
        "startChar": 8,
        "endLine": 421,
        "endChar": 46
      },
      "revId": "87a4a2d83d89429c0a3963436aac76c083d83c8e",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "fe702349_5a430268",
        "filename": "ibm_vpd_app.cpp",
        "patchSetId": 9
      },
      "lineNbr": 434,
      "author": {
        "id": 1000386
      },
      "writtenOn": "2022-01-21T04:44:14Z",
      "side": 1,
      "message": "Not sure on severity, if discussed already then no issues, else please check",
      "range": {
        "startLine": 434,
        "startChar": 38,
        "endLine": 434,
        "endChar": 58
      },
      "revId": "87a4a2d83d89429c0a3963436aac76c083d83c8e",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "7788157a_98e0f217",
        "filename": "ibm_vpd_app.cpp",
        "patchSetId": 9
      },
      "lineNbr": 1440,
      "author": {
        "id": 1000362
      },
      "writtenOn": "2022-01-24T05:21:42Z",
      "side": 1,
      "message": "same here. your cout statement is contradicting with the condition",
      "range": {
        "startLine": 1440,
        "startChar": 8,
        "endLine": 1440,
        "endChar": 46
      },
      "revId": "87a4a2d83d89429c0a3963436aac76c083d83c8e",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}